src/main/java/com/tileman/TilemanModeWorldMapOverlay.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2019, Benjamin <https://github.com/genetic-soybean>
+ * Copyright (c) 2020, Bram91 <https://github.com/Bram91>
```

Comment:
nit: you can probably add your name here since I assume you made some changes to this file to handle dynamic sizing :)